source/includes/documentation/_vue_js_elements.md

```diff
@@ -2,28 +2,62 @@
 
 ### 5.1 Alerts
 
-[Vuetify documentation page](https://vuetifyjs.com/en/components/alerts/)
+[Vuetify documentation page](https://v2.vuetifyjs.com/en/components/alerts/)
+
+- __Java__: _com.epam.jdi.light.vuetify.elements.common.Alert.java_
 
 ```java
-  @Test
-  public void alertsWithPropsHaveProperCssProps(){
-      redAlert.has().text("I'm an alert with a top border and red color");
-      redAlert.has().css("background-color", Colors.RED_LIGHTEN_2.value());
-      redAlert.has().cssClass("v-alert--border-top");
-  }
+    //@FindBy(css = ".v-alert")
+    @UI(".v-alert")
+    public static Alert alert;
+
+    @Test
```

Comment:
all tests should have descriptions :) that's our team agreement :)

Reply:
Fixed